bundles/FileExplorerBundle/public/js/startup.js

```diff
@@ -0,0 +1,44 @@
+pimcore.registerNS("pimcore.settings.fileexplorer");
+
+pimcore.settings.fileexplorer = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.pimcoreReady, this.pimcoreReady.bind(this));
+    },
+
+
+    pimcoreReady: function(e) {
```

Comment:
Please merge the latest changes from 11.x with the Glossarybundle and update the event listeners accordingly.
Checkout the startup.js from the glossary bundle https://github.com/pimcore/pimcore/blob/11.x/bundles/GlossaryBundle/public/js/startup.js

Please also make sure to update the namespaces of the js files like `pimcore.registerNS("pimcore.bundle.glossary.startup");` and `pimcore.bundle.glossary.startup = Class.create({`

Please also mark every js file as private as you can see in the startup.js from glossary.

A priority for the menu items should also be provided. 

See also the settings.js from the glossary for more info https://github.com/pimcore/pimcore/blob/11.x/bundles/GlossaryBundle/public/js/settings.js